src/Aspire.ProjectTemplates/templates/aspire-apphost/8.2/Properties/launchSettings.json

```diff
@@ -10,8 +10,8 @@
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development",
         "DOTNET_ENVIRONMENT": "Development",
-        "DOTNET_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21000",
-        "DOTNET_RESOURCE_SERVICE_ENDPOINT_URL": "https://localhost:22000"
+        "ASPIRE_DASHBOARD_OTLP_ENDPOINT_URL": "https://localhost:21000",
```

Comment:
does it make sense to update the old templates? Where the new env var names won't be respected?

Reply:
didn't mean to, will roll that back.